kubestash/__init__.py

```diff
@@ -462,6 +643,34 @@ def cmd_daemon(args):
         time.sleep(args.interval)
 
 
+def cmd_daemonall(args):
+    args.force = True
+
+    client, shard_iterator = get_stream_client(args)
+
+    if args.verbose:
+        print("checking DynamoDB Stream for changes...")
+
+    response = client.get_records(ShardIterator=shard_iterator, Limit=10)
+
+    while True:
+        shard_iterator = response['NextShardIterator']
+
+        if args.verbose:
+            print("checking DynamoDB Stream for changes...")
+
+        response = client.get_records(ShardIterator=shard_iterator, Limit=10)
+
+        if len(response['Records']) > 0:
+            key = response['Records'][0]['dynamodb']['Keys']['name']['S']
```

Comment:
This is still suboptimal, but will work as long as you're doing 1 update/interval. Will change to a loop on all records.

Also, just to note: this works with "KEY" type streaming enabled